src/main/java/Jonathan.java

```diff
@@ -30,24 +30,59 @@ public static void main(String[] args) {
                 boolean isDone = command.split(" ")[0].equals("mark");
                 int taskNum = Integer.parseInt(command.split(" ")[1]);
                 Task task = list[taskNum-1];
-
                 changeStatus(task, isDone);
 
-            } else { // add Task
-                list[taskCounter] = new Task(command);
+            } else if (command.split(" ")[0].equals("todo")) {
+                String description = command.substring(command.indexOf(" ") + 1);
+                Todo todo = new Todo(description);
+                list[taskCounter] = todo;
+                taskCounter += 1;
+                addedMessage(todo, taskCounter);
+
+            } else if (command.split(" ")[0].equals("deadline")) {
+                String description = command.substring(command.indexOf(" ") + 1, command.indexOf(" /by"));
+                String by = command.substring(command.indexOf("/by") + 4);
+                Deadline deadline = new Deadline(description, by);
+                list[taskCounter] = deadline;
+                taskCounter += 1;
+                addedMessage(deadline, taskCounter);
+
+            } else if (command.split(" ")[0].equals("event")) {
+                String description = command.substring(command.indexOf(" ") + 1, command.indexOf(" /from"));
+                String start = command.substring(command.indexOf("/from") + 6, command.indexOf(" /to"));
+                String end = command.substring(command.indexOf("/to") + 4);
+                Event event = new Event(description, start, end);
+                list[taskCounter] = event;
                 taskCounter += 1;
-                System.out.println(line + "\n    added: " + command + "\n" + line + "\n");
+                addedMessage(event, taskCounter);
+
+            } else { // add Task
+                System.out.println("    ____________________________________________________________\n" +
+                        "    Wrong input, please type it correctly!\n" +
+                        "    ____________________________________________________________\n");
             }
         }
     }
 
+    /*
+    Method to print the added message to the user after prompting input
```

Comment:
```suggestion
   * Method to print the added message to the user after prompting input
```

Perhaps you could add a * in front of this comment? I notice a similar issue in other places.